src/analysis/Analyses.ts

```diff
@@ -632,7 +637,7 @@ export const BaseAnalyses = (
           tool,
           namespace: workspaceInfo.namespace,
           workspaceName: workspaceInfo.name,
-          canWrite,
+          canWrite: canWrite(accessLevel),
```

Comment:
It's interesting, I would've changed it so that `canWrite(accessLevel)` was computed once, such as after `analysisPlaygroundLink`, instead of calling the function more than once.

Reply:
I missed the `_.map` and didn't see this was in a loop. Thank you for catching.